src/components/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+export default function App() {
+  return (
+    <div className="container">
+      <Display result={0} />
```

Comment:
- No need to pass zero here since you have a default value of zero in the Display component

Reply:
Done!